src/redist/targets/GenerateLayout.targets

```diff
@@ -212,26 +213,40 @@
       </BundledInstallerComponent>
 
       <BundledInstallerComponent Include="DownloadedAspNetCoreSharedFxInstallerFile"
-                                 Condition="'$(SkipBuildingInstallers)' != 'true' AND '$(DownloadedAspNetCoreSharedFxInstallerFileName)' != '' And '$(InstallerExtension)' != '' And !$(Architecture.StartsWith('arm'))">
+                                 Condition="'$(InstallerExtension)' != '.pkg' And '$(SkipBuildingInstallers)' != 'true' And '$(InstallerExtension)' != '' And !$(Architecture.StartsWith('arm'))">
```

Comment:
double check, we counted .deb and .rpm in this case?

Reply:
Yes, that is correct - and excluded the .pkg installers as the Asp.NET does not build native installers for Mac.